### ISSUE TITLE
Quesiton: Is the training time display accurate?

### ISSUE THREAD

**Issue by tjoymeed** [volcengine/verl#1393]
Hi all, 

Is the training time counter accurate? Or at least the time counter for past history should be accruate, am I right?

Looking at the following stats:

It shows that Step 1 takes about 10:12 minutes.

But the actual waiting time is far more than this: my feeling is that it is about 1 hour on my 8xA100 80G.

What might be the cause for this?

Thanks a lot!

--------------------------------

Training Progress:   0%|          | 0/4665 [00:00<?, ?it/s]
(WorkerDict pid=2127084) /usr/local/lib/python3.10/dist-packages/torch/distributed/fsdp/fully_sharded_data_parallel.py:690: FutureWarning: FSDP.state_dict_type() and FSDP.set_state_dict_type() are being deprecated. Please use APIs, get_state_dict() and set_state_dict(), which can support different parallelisms, FSDP1, FSDP2, DDP. API doc: https://pytorch.org/docs/stable/distributed.checkpoint.html#torch.distributed.checkpoint.state_dict.get_state_dict .Tutorial: https://pytorch.org/tutorials/recipes/distributed_checkpoint_recipe.html .
(WorkerDict pid=2127084)   warnings.warn(
(TaskRunner pid=2126429) step:0 - val-aux/math_dapo/reward/mean@32:nan - val-aux/math_dapo/reward/std@32:nan - val-aux/math_dapo/reward/best@2/mean:nan - val-aux/math_dapo/reward/best@2/std:nan - val-aux/math_dapo/reward/worst@2/mean:nan - val-aux/math_dapo/reward/worst@2/std:nan - val-aux/math_dapo/reward/maj@2/mean:nan - val-aux/math_dapo/reward/maj@2/std:nan - val-aux/math_dapo/reward/best@4/mean:nan - val-aux/math_dapo/reward/best@4/std:nan - val-aux/math_dapo/reward/worst@4/mean:nan - val-aux/math_dapo/reward/worst@4/std:nan - val-aux/math_dapo/reward/maj@4/mean:nan - val-aux/math_dapo/reward/maj@4/std:nan - val-aux/math_dapo/reward/best@8/mean:nan - val-aux/math_dapo/reward/best@8/std:nan - val-aux/math_dapo/reward/worst@8/mean:nan - val-aux/math_dapo/reward/worst@8/std:nan - val-aux/math_dapo/reward/maj@8/mean:nan - val-aux/math_dapo/reward/maj@8/std:nan - val-aux/math_dapo/reward/best@16/mean:nan - val-aux/math_dapo/reward/best@16/std:nan - val-aux/math_dapo/reward/worst@16/mean:nan - val-aux/math_dapo/reward/worst@16/std:nan - val-aux/math_dapo/reward/maj@16/mean:nan - val-aux/math_dapo/reward/maj@16/std:nan - val-aux/math_dapo/reward/best@32/mean:nan - val-aux/math_dapo/reward/best@32/std:nan - val-aux/math_dapo/reward/worst@32/mean:nan - val-aux/math_dapo/reward/worst@32/std:nan - val-aux/math_dapo/reward/maj@32/mean:nan - val-aux/math_dapo/reward/maj@32/std:nan - val-aux/math_dapo/score/mean@32:-0.879 - val-aux/math_dapo/score/std@32:0.137 - val-aux/math_dapo/score/best@2/mean:-0.832 - val-aux/math_dapo/score/best@2/std:0.139 - val-aux/math_dapo/score/worst@2/mean:-0.932 - val-aux/math_dapo/score/worst@2/std:0.093 - val-aux/math_dapo/score/maj@2/mean:-0.880 - val-aux/math_dapo/score/maj@2/std:0.138 - val-aux/math_dapo/score/best@4/mean:-0.778 - val-aux/math_dapo/score/best@4/std:0.131 - val-aux/math_dapo/score/worst@4/mean:-0.963 - val-aux/math_dapo/score/worst@4/std:0.054 - val-aux/math_dapo/score/maj@4/mean:-0.867 - val-aux/math_dapo/score/maj@4/std:0.101 - val-aux/math_dapo/score/best@8/mean:-0.720 - val-aux/math_dapo/score/best@8/std:0.124 - val-aux/math_dapo/score/worst@8/mean:-0.986 - val-aux/math_dapo/score/worst@8/std:0.035 - val-aux/math_dapo/score/maj@8/mean:-0.864 - val-aux/math_dapo/score/maj@8/std:0.061 - val-aux/math_dapo/score/best@16/mean:-0.661 - val-aux/math_dapo/score/best@16/std:0.096 - val-aux/math_dapo/score/worst@16/mean:-0.997 - val-aux/math_dapo/score/worst@16/std:0.016 - val-aux/math_dapo/score/maj@16/mean:-0.865 - val-aux/math_dapo/score/maj@16/std:0.026 - val-aux/math_dapo/score/best@32/mean:-0.626 - val-aux/math_dapo/score/best@32/std:0.056 - val-aux/math_dapo/score/worst@32/mean:-1.000 - val-aux/math_dapo/score/worst@32/std:0.003 - val-aux/math_dapo/score/maj@32/mean:-0.866 - val-aux/math_dapo/score/maj@32/std:0.006 - val-core/math_dapo/acc/mean@32:0.060 - val-aux/math_dapo/acc/std@32:0.068 - val-aux/math_dapo/acc/best@2/mean:0.084 - val-aux/math_dapo/acc/best@2/std:0.069 - val-aux/math_dapo/acc/worst@2/mean:0.034 - val-aux/math_dapo/acc/worst@2/std:0.046 - val-aux/math_dapo/acc/maj@2/mean:0.060 - val-aux/math_dapo/acc/maj@2/std:0.069 - val-aux/math_dapo/acc/best@4/mean:0.111 - val-aux/math_dapo/acc/best@4/std:0.066 - val-aux/math_dapo/acc/worst@4/mean:0.018 - val-aux/math_dapo/acc/worst@4/std:0.027 - val-aux/math_dapo/acc/maj@4/mean:0.067 - val-aux/math_dapo/acc/maj@4/std:0.050 - val-aux/math_dapo/acc/best@8/mean:0.140 - val-aux/math_dapo/acc/best@8/std:0.062 - val-aux/math_dapo/acc/worst@8/mean:0.007 - val-aux/math_dapo/acc/worst@8/std:0.017 - val-aux/math_dapo/acc/maj@8/mean:0.068 - val-aux/math_dapo/acc/maj@8/std:0.030 - val-aux/math_dapo/acc/best@16/mean:0.169 - val-aux/math_dapo/acc/best@16/std:0.048 - val-aux/math_dapo/acc/worst@16/mean:0.002 - val-aux/math_dapo/acc/worst@16/std:0.008 - val-aux/math_dapo/acc/maj@16/mean:0.067 - val-aux/math_dapo/acc/maj@16/std:0.013 - val-core/math_dapo/acc/best@32/mean:0.187 - val-core/math_dapo/acc/best@32/std:0.028 - val-aux/math_dapo/acc/worst@32/mean:0.000 - val-aux/math_dapo/acc/worst@32/std:0.001 - val-core/math_dapo/acc/maj@32/mean:0.067 - val-core/math_dapo/acc/maj@32/std:0.003
(TaskRunner pid=2126429) list(reward_extra_infos_dict.keys())=['score', 'acc', 'pred']
(TaskRunner pid=2126429) 
Training Progress:   0%|          | 1/4665 [10:12<793:07:13, 612.19s/it]

**Comment by nomadlx** [comment]
I feel that there is a problem with this time count. It only calculates the time of the last generation (gen), but in reality, the generation operation may have been run multiple times before filling the batch.

**Comment by Kirrito-k423** [comment]
### Scheduled issue cleanup / 定时 issue 清理

中文：该 issue 命中定时清理规则：最近一次非当前运行账号的公开回复或引用已经超过 6 个月。因此按定时清理标准先关闭。若作者或社区成员仍能提供新的复现、证据或需求背景，欢迎继续评论请求重开，或创建带有最新信息的新 issue。

证据：
- Issue: https://github.com/verl-project/verl/issues/1393
- 最近一次非当前运行账号的公开回复或公开引用是 `2025-05-08T05:23:36Z` UTC（comment by @nomadlx），截至 2026-07-05 已超过 6 个月。
- 已检查 comments 来源：https://api.github.com/repos/verl-project/verl/issues/1393/comments
- 已检查 timeline 来源：https://api.github.com/repos/verl-project/verl/issues/1393/timeline

English: This issue matches the scheduled cleanup rule: the latest public reply or reference by an account other than the current runner is older than 6 months. Therefore, this is being closed under the scheduled cleanup policy. If the author or community members can still provide a fresh reproduction, evidence, or requirement context, please comment to request reopening or open a new issue with updated information.

Evidence:
- Issue: https://github.com/verl-project/verl/issues/1393
- The latest public reply or public reference from an account other than the current runner was `2025-05-08T05:23:36Z` UTC (comment by @nomadlx), which is more than 6 months before 2026-07-05.
- Checked comments source: https://api.github.com/repos/verl-project/verl/issues/1393/comments
- Checked timeline source: https://api.github.com/repos/verl-project/verl/issues/1393/timeline

This response was generated by a scheduled Codex GPT5.5 issue processing task.
